client/ayon_core/cli.py

```diff
@@ -59,9 +59,9 @@ def main_cli(ctx, *_args, **_kwargs):
         if os.getenv("AYON_HEADLESS_MODE") == "1":
             print(ctx.get_help())
             sys.exit(0)
-        else:
-            ctx.params.pop("project")
-            ctx.forward(tray)
+
+        ctx.params.pop("project")
+        ctx.forward(tray)
```

Comment:
We should not start tray in headless mode.
```suggestion
        else:
            ctx.params.pop("project")
            ctx.forward(tray)
```